client/src/landing/WhatIsRenku/WhatIsRenku.tsx

```diff
@@ -0,0 +1,406 @@
+        <p>
+          All you need to run a RenkuLab project is a browser! Choose from our
+          curated compute environments to get started quickly. We take care of
+          containerization for you.
+        </p>
+        <TryOutSessionBtn
+          projectPath={projectPath}
+          type="autostart"
+          btnTitle="Try out a session"
+        />
+      </div>
+    </div>
+  );
+};
+
+const ComputeFeatSection = ({ projectPath }: WhatIsRenkuProps) => {
+  return (
+    <div id={styles.computeFeatContainer}>
+      <div className={styles.computeFeatGraph}>
+        <img
+          src={compute_Graphic}
+          alt="Compute Options Graph"
+          loading={"lazy"}
+        />
+      </div>
+      <div className={styles.computeFeatTitle}>
+        <h3>One project, many compute options</h3>
+        <p>
+          Scaling up your project is as simple as switching your session
+          resource class- everything else about your project stays the same. No
+          need to keep multiple compute environments in sync to scale up.
+        </p>
+      </div>
+      <div className={styles.computeFeatContain}>
+        <TryOutSessionBtn
+          projectPath={projectPath}
+          type="new"
+          btnTitle="Try out a session"
+        />
+      </div>
+      <div className={cx(["d-flex", "gap-3", styles.computeFeatOtherLink])}>
+        <div className="mt-1">
+          <Send size={30} />
+        </div>
+        <div className={styles.computeFeatSubtext}>
+          <span className="fw-bold d-block">
+            Want to take your Renku project to the next level?
+          </span>
+          <ExternalLink
+            className={"text-black"}
+            role="link"
+            id="computeFeat_link"
+            url={`mailto:${RenkuContactEmail}?subject=Renkulab%20compute%20resources`}
+            title="Contact us about larger compute resources on RenkuLab"
+          />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const ConnectFeatSection = () => {
+  return (
+    <div id={styles.connectFeatContainer}>
+      <div className={styles.connectFeatGraph}>
+        <img src={SSH_Graphic} loading="lazy" />
+      </div>
+      <div className={styles.connectFeatTitle}>
+        <h3>Connect from anywhere</h3>
+      </div>
+      <div className={styles.connectFeatContain}>
+        <p>
+          Connect to RenkuLab sessions from the comfort of your local terminal
+          or favorite IDE. Or, run your Renku project locally without RenkuLab
+          at all.
+        </p>
+        <p>
+          No need to keep collaborator’s environments in sync- collaborators can
+          open the same environment.
+        </p>
+        <div className={cx([styles.featDocLinks, "flex-wrap"])}>
+          <TerminalFill size={25} />
+          <div>
+            Try out the{" "}
+            <ExternalLink
+              color=""
+              role="link"
+              id="shareFeat_btn"
```

Comment:
```suggestion
              id="shareFeatBtn"
```
...or `share-feat-btn`
Mind this should be replaced also in other places on this page.